### DISCUSSION TITLE
Error Message on my Code

### DISCUSSION THREAD

**Discussion by Kay0978** [Cyfrin/foundry-full-course-cu#2994] [General]
Hello, I am experiencing a bug error on this line of my code, please kindly help check it out:

//view, pure
        function retrieve () public view return (unit256){
               return myFavoriteNumber;

**Comment by heheboii11** [comment]
Use ```returns``` instead of ```return```
you code should look like this 
```solidity
function retrieve () public view returns (unit256) {
return myFavoriteNumber;
}

```

**Reply by EngrPips** [reply]
Great Heads-up
